### PR TITLE
Demo apps

### PR DESCRIPTION
**Pull request by walterjgsp**

### CODE REVIEW

**Review by mcorner — COMMENTED** [review]
The android-demo should be in android/demo

File: examples/android-demo/README
Comment (mcorner) [inline examples/android-demo/README:1]: README.md

File: examples/android-demo/README [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Demo Android App
+
+This folder holds the Android app used to demo an integration with reCaptcha mobile
```
Comment (mcorner) [inline examples/android-demo/README:3]: reCAPTCHA

File: examples/android-demo/README
Comment (walterjgsp) [reply examples/android-demo/README:1]: Done

File: examples/android-demo/README (hunk 1 above)
Comment (walterjgsp) [reply examples/android-demo/README]: Done

File: examples/server/main.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import time
+import cgi
+import json
+import requests
+
+
+DEFAULT_HOST_NAME = "localhost"
+DEFAULT_SERVER_PORT = 8080
+USER_GUARD_CLOUD_PROJECT_NUMBER = "USER_GUARD_CLOUD_PROJECT_NUMBER"
+USER_GUARD_API_KEY = "USER_GUARD_API_KEY"
```
Comment (mcorner) [inline examples/server/main.py:25]: This isn't the public name.  Just use API_KEY and CLOUD_PROJECT_NUMBER

File: examples/server/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+To install all the dependencies of this server please run:
+
+```
+pip3 install -r requirements.txt
+```
+
+## How to use it
+
+Change the value of the fields `USER_GUARD_CLOUD_PROJECT_NUMBER` and `USER_GUARD_API_KEY` with
```
Comment (mcorner) [inline examples/server/README.md:16]: Change this to remove USER_GUARD

File: examples/ios/demo/RecaptchaDemo/RecaptchaDemoUITests/RecaptchaDemoUITests.swift [hunk 4]
```diff
@@ -0,0 +1,41 @@
+//
+//  RecaptchaDemoUITests.swift
+//  RecaptchaDemoUITests
+//
+//  Created by Walter Pinto on 2024-03-27.
```
Comment (mcorner) [inline examples/ios/demo/RecaptchaDemo/RecaptchaDemoUITests/RecaptchaDemoUITests.swift:5]: Find all of these and replace with a Google copyright/license.

File: examples/server/main.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+USER_GUARD_FULL_URL = f"https://recaptchaenterprise.googleapis.com/v1/projects/{USER_GUARD_CLOUD_PROJECT_NUMBER}/assessments?key={USER_GUARD_API_KEY}"
```
Comment (mcorner) [inline examples/server/main.py:27]: change to URL

File: examples/ios/demo/RecaptchaDemo/RecaptchaDemoUITests/RecaptchaDemoUITests.swift (hunk 4 above)
Comment (walterjgsp) [reply examples/ios/demo/RecaptchaDemo/RecaptchaDemoUITests/RecaptchaDemoUITests.swift:5]: Done

File: examples/server/main.py (hunk 5 above)
Comment (walterjgsp) [reply examples/server/main.py]: Done

**Review by mcorner — APPROVED** [review]

**Pull request merged**